packages/calcite-components/src/components/button/button.e2e.ts

```diff
@@ -644,4 +644,27 @@ describe("calcite-button", () => {
     expect(button1.textContent.length).toBeLessThan(longText.length);
     expect(button1.getAttribute("title")).toEqual(longText);
   });
+
+  it("should set aria-expanded attribute on shadowDOM element when used as trigger", async () => {
```

Comment:
Could you test if the `accessible` helper picks this up (also checking that it fails when the ARIA attribute isn't set on the internal button)? If so, I'd suggest using that to simplify this test.

Reply:
`accessible` helper is unable to pick this one up.